integration-tests/mail/src/test/java/org/apache/camel/quarkus/component/mail/MailTest.java

```diff
@@ -45,27 +70,89 @@
             + "Hello attachment!"
             + "${delimiter}--\r\n";
 
-    @Test
-    public void testSendAsMail() {
+    @BeforeEach
+    public void beforeEach() {
+        // Configure users
+        Config config = ConfigProvider.getConfig();
+        String userJson = String.format("{ \"email\": \"%s\", \"login\": \"%s\", \"password\": \"%s\"}", EMAIL_ADDRESS,
+                USERNAME, PASSWORD);
         RestAssured.given()
-                .contentType(ContentType.TEXT)
-                .body("Hi how are you")
-                .post("/mail/route/mailtext")
+                .contentType(ContentType.JSON)
+                .body(userJson)
+                .post("http://localhost:" + config.getValue("mail.api.port", Integer.class) + "/api/user")
                 .then()
                 .statusCode(200);
 
-        RestAssured.given()
-                .get("/mock/{username}/size", "james@localhost")
+        try {
+            Thread.sleep(3_000);
```

Comment:
Is the `Thread.sleep` necessary?

Reply:
oh I forgot that code from my local testing, will remove it.

Reply:
done